### PR TITLE
Set up basic testing structure for JS.

### PR DESCRIPTION
**Pull request by tblanshard**
Add basic JavaScript testing using the Jest framework.
(Tests that the JavaScript functions correctly create the graph data).

@andreeanica16 @iht @juliocc - thank you :)

### CODE REVIEW

**Review by juliocc — APPROVED** [review]
lgtm

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -228,6 +234,7 @@ function formatURLs(url, parameters) {
 }
 
 function getTotalCosts(aggregated){
+  console.log(aggregated)
```
Comment (andreeanica16) [inline src/main/webapp/script.js:239]: I think you should erase this

File: src/main/webapp/script.js [hunk 2]
```diff
@@ -102,10 +102,16 @@ function setUpGraphs() {
...
+      drawPieChart(getTotalCosts(jobData), 'Total Cost of Jobs Per Category', 'totalCost-container')
+    );
+    google.charts.setOnLoadCallback(
+      drawPieChart(getAverageCosts(jobData), 'Average Cost of Jobs Per Category', 'averageCost-container')
+    );
     google.charts.setOnLoadCallback(getDailyView(jobData));
-    google.charts.setOnLoadCallback(getFailedJobs(jobData));
+    google.charts.setOnLoadCallback(
+      drawPieChart(getFailedJobs(jobData), 'Total Number of Failed Jobs Per Category', 'failedJobs-container')
+    );
```
Comment (andreeanica16) [inline src/main/webapp/script.js]: Maybe you could have a function like this
`function setGraphOnLoad(data, string) {  
  google.charts.setOnLoadCallback(  
      drawPieChart(data, string);
    );
}`

You would have your code more modularized.

File: src/main/webapp/script.js (hunk 1 above)
Comment (tblanshard) [reply src/main/webapp/script.js]: Good point! All comments have been removed in 93be99b.

File: src/main/webapp/script.js (hunk 2 above)
Comment (tblanshard) [reply src/main/webapp/script.js]: Good idea :D Changed in 93be99b.

**Review by andreeanica16 — APPROVED** [review]
LGTM

**Pull request merged**